blocks/library/latest-posts/index.php

```diff
@@ -13,13 +13,18 @@
  * @return string Returns the post content with latest posts added.
  */
 function gutenberg_render_block_core_latest_posts( $attributes ) {
-	$recent_posts = wp_get_recent_posts( array(
-		'numberposts' => $attributes['postsToShow'],
-		'post_status' => 'publish',
-		'order'       => $attributes['order'],
-		'orderby'     => $attributes['orderBy'],
-		'category'    => $attributes['categories'],
-	) );
+	$recent_posts = wp_get_recent_posts(
```

Comment:
I think this should include:

`/** This filter is documented in wp-includes/widgets/class-wp-widget-recent-posts.php */`

Reply:
:+1: